os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c

```diff
@@ -1687,6 +1687,7 @@ T_APP_RESULT ble_tizenrt_scatternet_gcs_client_callback(T_CLIENT_ID client_id, u
                         }
                     } else {
                         debug_print("Memory allocation failed \n");
+                        os_mem_free(notify_result);
```

Comment:
Could you split this as different commit? Because this is the change on RTL8721CSM, not RTL8720E.

Reply:
Left out for now